release-notes/README.md

```diff
@@ -4,17 +4,15 @@ The following [.NET releases](../releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 7](7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-3/) | [Preview][policies] | [7.0 Preview 3][7.0 Preview 3] | May 14, 2024 |
-| [.NET 6](6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS][policies] | [6.0.4][6.0.4]  | November 12, 2024 |
-| [.NET 5](5.0/README.md) | [November 10, 2020](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/) | [Current][policies] | [5.0.16][5.0.16] | May 10, 2022 |
-| [.NET Core 3.1](3.1/README.md) | [December 3, 2019](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/) | [LTS][policies] | [3.1.24][3.1.24] | December 13, 2022 |
+| [.NET 7](7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) | [Preview][policies] | [7.0 Preview 4][7.0 Preview 4] | May 14, 2024 |
```

Comment:
It's a bit misleading saying .NET 7 is currently supported when it's still in preview and not currently supported right?